src/ray/protobuf/direct_actor.proto

```diff
@@ -7,10 +7,12 @@ import "src/ray/protobuf/common.proto";
 message ReturnObject {
   // Object ID.
   bytes object_id = 1;
+  // If set, indicates the data is in plasma instead of inline.
+  bool in_plasma = 2;
```

Comment:
add a comment that the below fields `data` and `metadata` would be empty in that case?